tests/backends/test_kafka.py

```diff
@@ -245,6 +256,10 @@ def test_consumer_polls_when_paused(self) -> None:
                 assert consumer.paused() == []
 
 
+class TestKafkaStreamsIncrementalRebalancing(TestKafkaStreams):
```

Comment:
unused?

Reply:
no, this actually re-declares all the tests in `TestKafkaStreams`, just re-running them with cooperative-sticky rebalancing